### ISSUE TITLE
Copy number prediction

### ISSUE THREAD

**Issue by mortonjt** [qiime2/q2-feature-classifier#108]
It looks like in the newer release of [rdp](https://rdp.cme.msu.edu/classifier/class_help.jsp#copynumber), copy number prediction / normalization is enabled.  This may be a good place to include this sort of functionality ( and it is super critical for applications such as picrust).

cc @gavinmdouglas

**Comment by nbokulich** [comment]
I agree, q2-feature-classifier would be an appropriate spot for this functionality.

However, I am [not sure if any of the 16S copy # normalizers are any good](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5828423/). Any recommendations?

Looks like all RDP does is weight the abundance of each taxon by (mean copy number of that taxon from rrnDB](https://github.com/rdpstaff/classifier/blob/master/samplefiles/rrnDBv4.2.2_copynumber.txt). That would be pretty easy to replicate here, and could be applied to any classifications if it is a separate method. Any volunteers?

Picrust has its own normalizer on board, so this would not be essential for supporting a picrust plugin.

**Comment by mortonjt** [comment]
Whoa, that is a little crazy, I didn't realize that those predictions were
that bad. Thanks for that link!

**Comment by nbokulich** [comment]
ha! This might be useful to have anyway — we have had a few users ask about it on the forum, so it might be the sort of thing that we provide but warn to proceed at a user's own risk.

Notably, it doesn't look like the RDP normalizer is benchmarked in that paper.

**Comment by gregcaporaso** [comment]
@nbokulich, @mortonjt - I'm going to close this since it sounds like we didn't have a great solution, and there hasn't been activity on this in a long time. If there is still interest, feel free to reopen.
